### DISCUSSION TITLE
How to Restrict Access to a Single IP Address

### DISCUSSION THREAD

**Discussion by aramhai** [devlikeapro/waha#370] [General]
I am using Docker on a Linux server with Ubuntu as the operating system.

I want to restrict access to Docker to only one IP address, so that the entire internet cannot access this site. I want only my server to be able to access the API. How do I do this?

**Comment by devlikepro** [comment]
You can use secure options available in WAHA Plus 
https://waha.devlike.pro/docs/how-to/security/

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
